assets/app/scripts/filters/util.js

```diff
@@ -117,6 +117,9 @@ angular.module('openshiftConsole')
           }
           unit += (amount === "1" ? "core" : "cores");
           break;
+        case "storage":
+          unit = unit.substring(0, 1);
```

Comment:
Ha, I wasn't aware we were using the gibitytes standards G (1000) and Gi (1024). Then I think we should reuse memory, we will end up having for example `GB` and `GiB`.